ffmpeg.yaml

```diff
@@ -53,13 +57,14 @@ pipeline:
 data:
   - name: libraries
     items:
-      libavcodec60: libavcodec
-      libavdevice60: libavdevice
-      libavfilter9: libavfilter
-      libavformat60: libavformat
-      libavutil58: libavutil
-      libswresample4: libswresample
-      libswscale7: libswscale
+      libavcodec61: libavcodec
+      libavdevice61: libavdevice
+      libavfilter10: libavfilter
+      libavformat61: libavformat
+      libavutil59: libavutil
+      libswresample5: libswresample
+      libswscale8: libswscale
+      libpostproc58: libpostproc
```

Comment:
ideally ffmpeg-dev package would depend on all the runtime libraries packages.

This would then avoid the need to encode them in the environment and runtime blocks of other packages.

Reply:
@xnox `ffmpeg-dev` already depends all of them, because it provides all .so symbolic links and it makes melange link it (like alpine [package](https://pkgs.alpinelinux.org/contents?branch=edge&name=ffmpeg%2ddev&repo=community&arch=x86_64&page=3))

You can see PKINFO from build. Do you think that we remove symbolic links from ffmpeg-dev and declare explicit runtime dependencies?

```console
[sdk] ❯ cat .PKGINFO
# Generated by melange v0.6.11-81-g256a03c
pkgname = ffmpeg-dev
pkgver = 7.0-r1
arch = aarch64
size = 1554089
origin = ffmpeg
pkgdesc = ffmpeg development headers
url =
commit = f384932378d31b5467a1d94ffa14a1b533d4640b
license = GPL-3.0-or-later AND LGPL-3.0-or-later
depend = so:libavcodec.so.61
depend = so:libavdevice.so.61
depend = so:libavfilter.so.10
depend = so:libavformat.so.61
depend = so:libavutil.so.59
depend = so:libpostproc.so.58
depend = so:libswresample.so.5
depend = so:libswscale.so.8
provides = pc:libavcodec=61.3.100
provides = pc:libavdevice=61.1.100
provides = pc:libavfilter=10.1.100
provides = pc:libavformat=61.1.100
provides = pc:libavutil=59.8.100
provides = pc:libpostproc=58.1.100
provides = pc:libswresample=5.1.100
provides = pc:libswscale=8.1.100
datahash = 1a28c1b013aa6089655d0657ac09be3fabf9d55982ff2bfa847615f60bc37ad3
```

Reply:
oh, that's great, in that case, i'd rather want you to remove explicit environment key `libswscale8` whenever `ffmpeg-dev` is already mentioned.

See my new single line comments "redundant".

Reply:
@xnox great, thanks you